tortoise/backends/psycopg/executor.py

```diff
@@ -26,3 +26,7 @@ async def _process_insert_result(
 
     def parameter(self, pos: int) -> Parameter:
         return Parameter("%s")
+
+    @classmethod
+    def parameterizer(cls) -> Parameterizer:
+        return Parameterizer(placeholder_factory=lambda _: "%s")
```

Comment:
psycopg decided to use non-standard placeholder for Postgres, so we have to account for that.